app/src/test/java/nerd/tuxmobil/fahrplan/congress/schedule/ConferenceTest.kt

```diff
@@ -1,8 +1,8 @@
 package nerd.tuxmobil.fahrplan.congress.schedule
 
+import com.google.common.truth.Truth.assertThat
```

Comment:
why not use the junit assertThat?
https://junit.org/junit4/javadoc/latest/org/junit/Assert.html#assertThat(java.lang.String,%20T,%20org.hamcrest.Matcher)
(nothing which needed to change here, a general question)

Reply:
:arrow_right: I prefer _Truth_ mostly because of the better test failure messages. Read on here: https://truth.dev